README.md

```diff
@@ -1,2 +1,54 @@
-# Hello-Microverse
-A Hello Microverse small project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> A simple  Hello Microverse small project this project includes just few files: the stylesheet file, the html file and other coplementary files 
+
+
+## Built With
+
+- html and css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+you mast have vs code or any other text editor
+install a web browser in your local compupter
+
```

Comment:
- [ ] [optional] It would be nice to have a description on how to clone and run your project locally.